sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/CosmosConfigSpec.scala

```diff
@@ -47,12 +47,12 @@ class CosmosConfigSpec extends UnitSpec {
   "Config Parser" should "parse default account AAD authentication credentials" in {
       val userConfig = Map(
           "spark.cosmos.accountEndpoint" -> "https://boson-test.documents.azure.com:443/",
-          "spark.cosmos.auth.type" -> "ServicePrinciple",
+          "spark.cosmos.auth.type" -> "ServicePrincipal",
```

Comment:
Test coverage for both correctly and incorrectly spelled should be maintained

Reply:
Well, in this case, wouldn't the "new" test coverage relate to this version and not an old version with the incorrectly spelled auth type ? Or would the test coverage ever run in an environment where the misspelled auth type exists, considering it is the same code base, perhaps Im not quite udnerstanding how it would ever properly validate the new and old spelling. Thoughts?